Nodejs/Product/TestAdapter/TestExecutorWorker.cs

```diff
@@ -0,0 +1,390 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.NetworkInformation;
+using System.Runtime.InteropServices;
+using System.Threading;
+using Microsoft.NodejsTools.TestAdapter.TestFrameworks;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.Adapter;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.Logging;
+using Microsoft.VisualStudioTools;
+using Microsoft.VisualStudioTools.Project;
+using Newtonsoft.Json;
+
+namespace Microsoft.NodejsTools.TestAdapter
+{
+    internal sealed class TestExecutorWorker
+    {
+        private static readonly Version Node8Version = new Version(8, 0);
+
+        //get from NodeRemoteDebugPortSupplier::PortSupplierId
+        private static readonly Guid NodejsRemoteDebugPortSupplierUnsecuredId = new Guid("{9E16F805-5EFC-4CE5-8B67-9AE9B643EF80}");
+
+        private readonly ManualResetEvent cancelRequested = new ManualResetEvent(false);
```

Comment:
>cancelRequested [](start = 42, length = 15)

I'm probably missing something, but this doesn't appear to be used for blocking.  If it's just a flag, consider using a simpler type.

Reply:
Sure, but don't really want to touch this at this point.